CHANGELOG.md

```diff
@@ -7,6 +7,31 @@ command line for details.
 ## [Unreleased]
 
 
+## [4.2] - 2019-11-13
+
+#### Summary
+
+- Now terminates on `SIGTERM` as can be caused by `docker stop` or `kubectl delete`
+- Add `--timeout` option to configure when to drop a request
+- Add `--custom-header` option that enables proxied requests to get additional headers attached
+- Support setting of the environment variable `CONFIGPROXY_AUTH_TOKEN` using a mounted file on the Docker image's container
+- Node version bumped from 10 to 12.13.0 in the Docker image
+- Various dependencies updated and security vulnerabilities patched
```

Comment:
Let's be careful about noting that vulnerabilities are patched. These are not vulnerabilities in operating CHP, but rather unused portions of packages in the npm dependency tree, mainly used in testing.

```suggestion
- Various dependencies updated, including addressing security advisories from `npm audit` which do not affect CHP security itself.
```